src/Markdown.hs

```diff
@@ -26,14 +16,23 @@ instance ToMarkup Markdown where
     -- | Sanitized by default
     toMarkup = handleError . markdownToHtml
 
-strippedText :: ToMarkup a => a -> Text
-strippedText = pack . runLA
-    (   arr toHtml
-    >>> arr renderMarkup
-    >>> hread
-    //> getText
-    >>. concat
-    )
+toPandoc :: Markdown -> Pandoc
+toPandoc m = either mempty id $ parse m
+  where
+    parse :: Markdown -> Either PandocError Pandoc
+    parse = readMarkdown readerOptions . unpack . unMarkdown
+
+plainText :: Markdown -> Text
+plainText = pack
+            . writePlain plainWriterOptions
+            . topDown removeInlineStyles
+            . toPandoc
+
+removeInlineStyles :: [Inline] -> [Inline]
+removeInlineStyles ((Emph lst) : xs) = removeInlineStyles $ lst ++ xs
+removeInlineStyles ((Strong lst) : xs) = removeInlineStyles $ lst ++ xs
+removeInlineStyles (x : xs) = x : removeInlineStyles xs
+removeInlineStyles [] = []
```

Comment:
That gets weird because `Emph` and `Strong` both have `[Inline]` as... payloads? Idk what the word for that is. But anyway, there's no good way to do `[Inline] -> Inline` (that I could find) so this seemed like the best solution (even though, yeah, a bit gnarly to read).